### DISCUSSION TITLE
Using rust webrtc in production in combination with flutter apps

### DISCUSSION THREAD

**Discussion by dscherdi** [webrtc-rs/webrtc#66] [General]
Hello!

I am interested in this library for implementing native WebRtc for Flutter apps.
In our Apps we would like to have cross-platform native functionality for conference calls, screen sharing etc. Main systems we are currently targeting are Windows/Linux/Mac. However I noticed that some API's are missing like Data and Media.

Would like to know if these packages are going to be implemented any time soon, and if more contribution is needed, how can one start implementing the specification for these API's.

Disclaimer:
I am totally new developer to WebRtc, but for business purposes we need this native functionality on our apps.

Looking forward to hearing from you. 
Best Regards

**Comment by rainliu** [comment]
@dscherdi, the WebRTC API level's crates, like PeerConnection, DataChannel and Media, are still not ready yet. Currently, we are focusing on the last piece of underlying protocols, SCTP. Once SCTP crate is ready, DataChannel API will be first implemented on top of SCTP.  Then, followed by PeerConnection crate.

**Reply by dscherdi** [reply]
Cool, seems like a nice plan. Do you also accept contributions ?

**Comment by rainliu** [comment]
Of course. Contribution and PR are welcome.
